prices/getawsprices.go

```diff
@@ -53,5 +37,99 @@ func GetPricesMapFromFile(filename string) (PricesMap, error) {
 	if err != nil {
 		return nil, err
 	}
-	return getPricesMap(bytes.NewReader(bs))
+	return populatePricesMap(bytes.NewReader(bs))
+}
+
+func DownloadPricesMap(url string) (PricesMap, error) {
+	if url == "" {
+		return PricesMap{}, nil
+	}
+
+	// TODO: working TLS
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+	client := &http.Client{Transport: tr}
+
+	res, err := client.Get(url)
+	if err != nil {
+		log.Error(err.Error())
+	}
+	defer res.Body.Close()
```

Comment:
I think res.Body is nil if err != nil on line 54. An error should probably be returned in the if statement above.